modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -48,6 +48,22 @@
 
   // Client properties
   private static final String CLIENT_PREFIX = FLUO_PREFIX + ".client";
+
+  /**
+   * @since 1.2.0
+   */
+  public static final String CLIENT_KERBEROS = CLIENT_PREFIX + ".kerberos";
```

Comment:
Since this is only used for HDFS and not for zookeeper or accumulo, I think the prop name should be `CLIENT_HDFS_KERBEROS = CLIENT_PREFIX + ".hdfs.kerberos"`